components/open-map-button.tsx

```diff
@@ -0,0 +1,25 @@
+import { SecondaryButton } from "./ui/button";
+
+import { MapIcon } from "@heroicons/react/outline";
+
+export function OpenMapButton({ address }: { address: string }) {
+  function btnClick(e: React.FormEvent<HTMLButtonElement>) {
+    e.currentTarget.blur();
+    window.open(`https://www.google.com/maps/search/?api=1&query=${address}`);
+  }
+
+  return (
+    <div className="flex items-start">
+      <SecondaryButton
+        aria-label="Buka Map"
```

Comment:
```suggestion
        aria-label="Buka Peta"
```
Please make it consistence with the button label